test/meta_transaction_test.ts

```diff
@@ -0,0 +1,734 @@
+                        expectedOrders: [validationResults.accepted[0].signedOrder],
+                        expectedPrice: price,
+                    });
+                    transaction = response.body.zeroExTransaction;
+                });
+
+                it.skip('submitting the quote is successful and money changes hands correctly', async () => {
+                    const makerAddress = validationResults.accepted[0].signedOrder.makerAddress;
+                    await weth.deposit().awaitTransactionSuccessAsync({ from: takerAddress, value: buyAmount });
+                    await weth
+                        .approve(contractAddresses.erc20Proxy, new BigNumber(buyAmount))
+                        .awaitTransactionSuccessAsync({ from: takerAddress });
+
+                    const startMakerWethBalance = await weth.balanceOf(makerAddress).callAsync();
+                    const startMakerZrxBalance = await zrx.balanceOf(makerAddress).callAsync();
+                    const startTakerWethBalance = await weth.balanceOf(takerAddress).callAsync();
+                    const startTakerZrxBalance = await zrx.balanceOf(takerAddress).callAsync();
+
+                    const signature = signZeroExTransaction(transaction, takerAddress);
+                    const route = constructRoute({
+                        baseRoute: `${META_TRANSACTION_PATH}/submit`,
+                    });
+                    const response = await httpPostAsync({
+                        route,
+                        body: {
+                            zeroExTransaction: transaction,
+                            signature,
+                        },
+                        headers: {
+                            '0x-api-key': config.WHITELISTED_API_KEYS_META_TXN_SUBMIT[0],
+                        },
+                    });
+                    expect(response.status).to.be.eq(HttpStatus.OK);
+                    expect(response.type).to.be.eq('application/json');
+
+                    const endMakerWethBalance = await weth.balanceOf(makerAddress).callAsync();
+                    const endMakerZrxBalance = await zrx.balanceOf(makerAddress).callAsync();
+                    const endTakerWethBalance = await weth.balanceOf(takerAddress).callAsync();
+                    const endTakerZrxBalance = await zrx.balanceOf(takerAddress).callAsync();
+                    expect(endMakerWethBalance).to.be.bignumber.eq(startMakerWethBalance.plus(sellAmount));
+                    expect(endMakerZrxBalance).to.be.bignumber.eq(startMakerZrxBalance.minus(buyAmount));
+                    expect(endTakerWethBalance).to.be.bignumber.eq(startTakerWethBalance.minus(sellAmount));
+                    expect(endTakerZrxBalance).to.be.bignumber.eq(startTakerZrxBalance.plus(buyAmount));
+                });
+            });
+
+            // TODO: There is a problem with this test case. It is currently throwing an `IncompleteFillError`
```

Comment:
We will be reworking these tests in the next meta-txn pr, so happy for these to be skipped for now.